readconf.h

```diff
@@ -20,7 +20,6 @@
 
 #define SSH_MAX_HOSTS_FILES	32
 #define MAX_CANON_DOMAINS	32
-#define PATH_MAX_SUN		(sizeof((struct sockaddr_un *)0)->sun_path)
```

Comment:
Why did you move this?

Reply:
It was only used internally in the corresponding C file. It is good to keep header files small, to avoid polluting the namespace and to avoid excess recompilation in case the define is changed.